### PR TITLE
Better mobile calendar setup explanation

### LINKED ISSUE #663: [BUG] TuiMobileCalendarDialog doesn't work
# 🐞 Bug report

### Description

I tried following [the docs](https://taiga-ui.dev/components/mobile-calendar/Setup), but the mobile calendar dialog doesn't appear when using `InputDate` or `InputDateRange`. The docs state it should work by default, is this not the case? Or am I missing something?

### Reproduction

I have an app with multiple `InputDate`s and `InputDateRange`s. I imported `TuiMobileCalendarDialogModule` in every module of my app, but `InputDate` and `InputDateRange` don't bring up a calendar window on mobile.

### Expected behavior

I expected it to work by default.

### Versions

-   OS: iOS, Android
-   Browser: Chrome, Safari and Brave
-   Angular: 12
-   Package: `@taiga-ui/addon-editor` v2.14.0

### PR DESCRIPTION
**Pull request by LuckeeDev**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

-   [ ] Bugfix
-   [ ] Feature
-   [ ] Refactoring (no functional changes, no api changes)
-   [x] Other: docs update

## What is the current behavior?

The [docs](https://taiga-ui.dev/components/mobile-calendar) for `TuiMobileCalendar` are unclear about the requirements for the component to work, leading to a difficult setup experience if one has never used Taiga UI.

Closes #663.

## What is the new behavior?

With the changes proposed, the docs explicitly state to add `TuiDialogModule` in order for the component to work.

## Does this PR introduce a breaking change?

-   [ ] Yes
-   [x] No

### CODE REVIEW

File: projects/demo/src/modules/components/mobile-calendar/mobile-calendar.template.html [hunk 1]
```diff
@@ -7,7 +7,9 @@
             <a tuiLink [routerLink]="['/components/input-date-range']"
                 >InputDateRange</a
             >
-            if <code>TuiMobileCalendarDialogModule</code> is imported
+            if <code>TuiMobileCalendarDialogModule</code> is imported (note: it
+            also needs <code>TuiDialogModule</code> to enable dialogs in the
+            app)
```
Comment (waterplea) [inline projects/demo/src/modules/components/mobile-calendar/mobile-calendar.template.html:12]: Let's also add that both need to be imported **once** in the **main module**.

File: projects/demo/src/modules/components/mobile-calendar/mobile-calendar.template.html [hunk 2]
```diff
@@ -106,8 +108,9 @@
         <ol class="b-demo-steps">
             <li>
                 <p i18n>
-                    Import <code>TuiMobileCalendarModule</code> into a module
-                    where you want to use our component
+                    Import <code>TuiDialogModule</code> and
+                    <code>TuiMobileCalendarModule</code>
+                    into a module where you want to use our component
```
Comment (waterplea) [inline projects/demo/src/modules/components/mobile-calendar/mobile-calendar.template.html:113]: And here

**Review by waterplea — CHANGES_REQUESTED** [review]
Thank you! Please see a little comment

**Comment by LuckeeDev** [comment]
@waterplea done! Let me know 😄

**Review by waterplea — APPROVED** [review]

**Pull request merged**